windowsforms/Sudoku/VB/sudoku/core/Game.vb

```diff
@@ -0,0 +1,58 @@
+﻿Public Class Game
+
+    Public Event ShowClues(grid()() As Integer)
+    Public Event ShowSolution(grid()() As Integer)
+
+    Private ReadOnly _hRow(8) As List(Of Integer)
+    Private ReadOnly _vRow(8) As List(Of Integer)
+    Private ReadOnly _threeSquare(8) As List(Of Integer)
+    Private ReadOnly _grid(8)() As Integer
+
+    Private _r As Random
+
+    Public Sub NewGame(rn As Random)
+        Me._r = rn
+        createNewGame()
+    End Sub
+
+    Private Sub initializeLists()
```

Comment:
This is still camelCase, it should be PascalCase

Reply:
VB does not have an .editorConfig file under Samples. I can guess what it should be and include in repo

Reply:
There is a .editorconfig file at the repository root, and I think there is an open issue to update it. You can update these to follow the convention manually.

Reply:
Someone updated it and my feature now complies with it.